yarn-project/aztec.js/src/wallet/account_manager.ts

```diff
@@ -48,7 +49,14 @@ export class AccountManager {
     public readonly salt: Salt,
   ) {}
 
-  static async create(wallet: Wallet, pxe: PXE, secretKey: Fr, accountContract: AccountContract, salt?: Salt) {
+  static async create(
+    wallet: Wallet,
+    pxe: PXE,
+    node: AztecNode,
```

Comment:
Eventually, this should not accept pxe.

Reply:
The account manager is only ever created by wallets and with the exception of embedded wallets never shared outside of it, so it is (mostly) fine.